tutorials/3d/introduction_to_global_illumination.rst

```diff
@@ -0,0 +1,344 @@
+
+       VoxelGI in action.
+
+- **SDFGI:** |average| Good reflections and indirect lighting, but beware of leaks and visible cascade shifts.
+
+  - GI level of detail varies depending on the distance
+    between the camera and surface.
+
+    Leaks can be reduced significantly by enabling the **Use Occlusion**
+    property. This has a small performance cost, but it often results in fewer
+    leaks compared to VoxelGI.
+
+    Cascade shifts may be visible when the camera moves fast. This can be made
+    less noticeable by adjusting the cascade sizes or using fog.
+
+    .. figure:: img/gi_sdfgi.png
+       :alt: SDFGI in action.
+
+       SDFGI in action.
+
+- **Screen-space indirect lighting (SSIL):** |average| Good *secondary* source of indirect lighting, but no reflections.
+
+  - SSIL is designed to be used as a complement to another GI technique such as
+    VoxelGI, SDFGI or LightmapGI. SSIL works best for small-scale details, but it
+    cannot provide accurate indirect lighting for large structures on its own.
+    SSIL can provide real-time indirect lighting in situations where other GI
+    techniques fail to capture small-scale details or dynamic objects.
+    Its screen-space nature will result in some artifacts,
+    especially when objects enter and leave the screen.
+    SSIL works with the viewport's emission buffer, which means that emissive decals
+    and custom shaders will result in indirect lighting being emitted
+    (as long as they're present on screen).
+
+    .. figure:: img/gi_ssil_only.png
+       :alt: SSIL in action (without any other GI technique). Notice the emissive lighting around the yellow box.
+
+       SSIL in action (without any other GI technique). Notice the emissive lighting around the yellow box.
+
+- **LightmapGI:** |good| Excellent indirect lighting, decent reflections (optional).
+
+  - This is the only technique where the number of light bounces
+    can be pushed above 2 (up to 16). When directional information
+    is enabled, spherical harmonics (SH) are used
+    to provide blurry reflections.
+
+    .. figure:: img/gi_lightmap_gi_indirect_only.png
+       :alt: LightmapGI in action. Only indirect lighting is baked here, but direct light can also be baked.
+
+       LightmapGI in action. Only indirect lighting is baked here, but direct light can also be baked.
+
+- **ReflectionProbe:** |average| Good reflections, but poor indirect lighting.
+
+  - Indirect lighting is a constant value spread throughout the probe,
+    essentially acting as local ambient lighting. Reflections and indirect
+    lighting are blended with other nearby probes.
+
+    .. figure:: img/gi_none_reflection_probe.png
+       :alt: ReflectionProbe in action (without any other GI technique). Notice the reflective sphere.
+
+       ReflectionProbe in action (without any other GI technique). Notice the reflective sphere.
+
+Real-time ability
+^^^^^^^^^^^^^^^^^
+
+- **VoxelGI:** |good| Fully real-time.
```

Comment:
I don't think the "|good|" and "|average|" labels here are helpful. Either a technique is realtime or it isn't. I would just remove the labels between the "| |" here and keep the "Fully real-time" "Semi real-time" labels

Reply:
The labels are only used to provide colored dots (`good` = green, `average` = yellow, `bad` = red). They're not visible as-is to the user.